funnel/views/project.py

```diff
@@ -609,7 +605,7 @@ def deregister(self):
     @requires_roles({'promoter'})
     def rsvp_list(self):
         return {
-            'project': self.obj,
+            'project': self.obj.current_access(),
             'going_rsvps': self.obj.rsvps_with(RSVP_STATUS.YES),
```

Comment:
Needs `'going_rsvps': [_r.current_access(datasets=('without_parent', 'related')) for _r in self.obj.rsvps_with(RSVP_STATUS.YES)],`